src/test/scenario-test.ts

```diff
@@ -10,7 +10,7 @@ require('source-map-support').install();
 
 @suite class Process {
     async runAz(directory: string, each: string) {
-        let cmd = `${__dirname}/../../` + "node_modules/.bin/autorest --version=3.0.6271 --az --use=" + `${__dirname}/../../` + " " + directory + "/configuration/readme.md --output-folder=" + directory + "/tmpoutput/src/" + each;
+        let cmd = `${__dirname}/../../` + "node_modules/.bin/autorest --version=3.0.6271 --az --use=" + `${__dirname}/../../` + " " + directory + "/configuration/readme.md --azure-cli-extension-folder=" + directory + "/tmpoutput/";
```

Comment:
>azure-cli-extension-folder [](start = 174, length = 26)

where is azure-cli-extension-folder used? in the code?

Reply:
this is the scenarios test command line. and azure-cli-extension-folder is at the right side of this line.